src/Theme.css

```diff
@@ -0,0 +1,68 @@
+/*Supported themes */
+.light-theme {
```

Comment:
i'm not an expert in CSS, what's the browser support for variables? is it universal at this point?

Reply:
CSS variables are widely supported. The only browser that doesn't support variables is Microsofts IE. However, support for the Internet Explorer is running out anyways. Do we need to support it?